### PR TITLE
Added test cases related to Youtube and Netflix as a part of WPE-502 and WPE-496

### PR DESCRIPTION
**Pull request by sandeep-vedam**
All the Youtube and Netflix cases are added as a part of WPE-502 and WPE-496

### CODE REVIEW

File: src/commonMethods/commonFunctions.js [hunk 1]
```diff
@@ -84,6 +85,39 @@ export const setWebKitUrl = function(URL) {
     .catch(err => err)
 }
 
+/**
+ * This function resumes or suspends Youtube plugin
+ * @param action
+ */
+export const youtubeActions = function(action) {
```
Comment (wouterlucas) [inline src/commonMethods/commonFunctions.js]: I'd rename this to youtubeChangeState

File: src/tests/youtube/Youtube_EME_001.test.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
+import {
+  pluginDeactivate,
+  webKitBrowserStartAndResume,
+  setWebKitUrl,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+import { AttachToLogs } from '../../commonMethods/remoteWebInspector'
+
+let url =
+  'http://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2016.html?test_type=encryptedmedia-test&enablewebm=off&command=run&tests=1,6,17,18'
```
Comment (wouterlucas) [inline src/tests/youtube/Youtube_EME_001.test.js]: Needs to be updated to the 2018 or 2019 eme tests: http://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2018.html?test_type=encryptedmedia-test&enablewebm=off&command=run&timestamp=1582733202787

or: http://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2019.html?test_type=encryptedmedia-test&enablewebm=off&command=run&timestamp=1582733202787

File: src/tests/youtube/Youtube_EME_002.test.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
+import {
+  pluginDeactivate,
+  webKitBrowserStartAndResume,
+  setWebKitUrl,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+import { AttachToLogs } from '../../commonMethods/remoteWebInspector'
+
+let url =
+  'https://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2017.html?test_type=encryptedmedia-test&command=run'
```
Comment (wouterlucas) [inline src/tests/youtube/Youtube_EME_002.test.js]: We can drop the 2017 test and only support the 2018 one below

File: src/tests/youtube/Youtube_EME_001.test.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
```
Comment (wouterlucas) [inline src/tests/youtube/Youtube_EME_001.test.js:11]: Also we need to only run the playready tests AFAIK

**Review by wouterlucas — CHANGES_REQUESTED** [review]

File: src/commonMethods/commonFunctions.js [hunk 5]
```diff
@@ -174,7 +210,6 @@ export const screenshot = async function() {
   let url = `http://${constants.host}:80/Service/Snapshot/Capture?${moment().valueOf()}`
   // create a new promise inside of the async function
   let bufferData = new Promise((resolve, reject) => {
-    //TODO : Replace _http by using this.$http helper
```
Comment (michielvandergeest) [inline src/commonMethods/commonFunctions.js:177]: Why was this todo comment removed?

File: src/tests/youtube/Youtube_EME_001.test.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/tests/youtube/Youtube_EME_001.test.js]: Hi @wouterlucas ,
In metromatic test, only 1,6,7,18 tests were run. Please let me know what tests to be run here.

File: src/commonMethods/commonFunctions.js (hunk 5 above)
Comment (sandeep-vedam) [reply src/commonMethods/commonFunctions.js:177]: This was removed by mistake. Reverting this line to the original.

File: src/tests/youtube/Youtube_EME_001.test.js (hunk 4 above)
Comment (sandeep-vedam) [reply src/tests/youtube/Youtube_EME_001.test.js:11]: Updated the URL to 2018

File: src/tests/youtube/Youtube_EME_002.test.js (hunk 3 above)
Comment (sandeep-vedam) [reply src/tests/youtube/Youtube_EME_002.test.js]: Updated the URL to 2019

File: src/commonMethods/commonFunctions.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/commonMethods/commonFunctions.js]: Renamed to youtubeChangeState

**Review by michielvandergeest — APPROVED** [review]

**Pull request merged**